scripts/github/setup/setup-github-project.sh

```
@@ -0,0 +1,714 @@
+- component: project-setup" \
+        --label "priority: high,type: epic,component: project-setup" \
+        --repo "$repo_owner/$repo_name")
+    
+    local epic_number=$(echo "$epic_issue" | grep -o '[0-9]*$')
+    
+    if [ -n "$epic_number" ]; then
+        print_success "Created project setup epic: #$epic_number"
+        
+        # Add to project if project ID exists
+        if [ -f ".github/project-id.txt" ]; then
+            local project_id=$(cat .github/project-id.txt)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)